.travis.yml

```diff
@@ -18,15 +23,15 @@ matrix:
     - php: 7.1
       env: SYMFONY_VERSION=3.3.*
     - php: 7.1
-      env: SYMFONY_VERSION=^3.4@dev
-  fast_finish: true
-  allow_failures:
+      env: SYMFONY_VERSION=3.4.*
     - php: 7.1
-      env: SYMFONY_VERSION=^3.4@dev
+      env: DEPS=dev SYMFONY_VERSION=4.0.*
+  fast_finish: true
 
 before_install:
   - composer self-update || true
   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
+  - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
```

Comment:
looking at https://github.com/symfony/symfony-docs/pull/8701/ this would be more elegant as `- if ! [ -z "$STABILITY" ]; then composer config minimum-stability ${STABILITY}; fi;` and use stability instead of DEPS so we can also do beta or whatever we need.

Reply:
Ok, sounds better :-)